### ISSUE TITLE
Some examples appear to be broken

### ISSUE THREAD

**Issue by jan-niestadt** [molleindustria/p5.play#218]
Looking into p5.play and it looks great!

I did notice a few examples that may be broken:
- [Using the virtual camera](https://molleindustria.github.io/p5.play/examples/index.html?fileName=camera.js)
- [Mouse events on sprites with camera](https://molleindustria.github.io/p5.play/examples/index.html?fileName=mouseCamera.js)
- [Game - Flappy Bird](https://molleindustria.github.io/p5.play/examples/index.html?fileName=flappyBird.js)

For all of these, I just see an empty canvas or static shape, but no moving sprites. Tried both Firefox and Chrome.

This message appears in the console:

```
Uncaught ReferenceError: cameraPop is not defined
    at Camera.off (p5.play.js:1788:6)
    at draw (eval at s (player.js:25:9), <anonymous>:34:10)
    at p5._main.default.redraw (p5.js:56238:19)
    at _draw (p5.js:48842:25)
```

**Comment by quinton-ashley** [comment]
oops yeah my mistake, easy fix though, thanks for pointing it out. Patching this now.

**Comment by quinton-ashley** [comment]
fixed in v2.2.3
